searchcore/src/vespa/searchcore/proton/server/initialize_threads_calculator.h

```diff
@@ -25,8 +26,7 @@ namespace proton {
      InitializeThreads _threads;
 
  public:
-     InitializeThreadsCalculator(const vespalib::string& base_dir,
-                                 uint32_t configured_num_threads);
+     InitializeThreadsCalculator(const HwInfo & hwInfo, const vespalib::string& base_dir, uint32_t configured_num_threads);
```

Comment:
Consider only passing in HwInfo::Cpu. That will also make setup in unit tests easier.